.github/actions/mint-token/action.yml

```diff
@@ -54,7 +54,7 @@ runs:
               body: REQUESTED_PERMISSIONS,
               headers: {
                 'Content-Type': 'application/json',
-                'X-OIDC-Token': idToken,
+                'X-GitHub-OIDC-Token': idToken,
```

Comment:
FWIW, we could just send both headers for awhile. That will get new minty and old minty versions.